### PR TITLE
Ticket #1096: Small content changes for Domain buttons

### LINKED ISSUE #1096: Django Admin: Update text and status messaging for "get status" button
### Issue Description

In Django admin, on the Domain page, there is a button to "get status."

- Update text on the "get status" button to read: "Get registry status"
- Update other buttons on this page to use sentence case, as well; "Manage domain" and ~"Delete registry domain"~ "Delete domain in registry"
- Update the status message that appears upon clicking the button to explain what status is being represented.

### Additional Context (optional)

Status message should read: ~"The registry statuses are ['serverTransferProhibited', 'inactive']. These statuses are from the EPP provider of the .gov registry."~
"The registry statuses are ['serverTransferProhibited', 'inactive']. These statuses are from the EPP provider of the .gov registry."

### Issue Links

_No response_

### PR DESCRIPTION
**Pull request by zandercymatics**
## Ticket

Resolves #1096 

## Changes

- Updated text on the "get status" button to read: "Get registry status"
- Changed the "Manage domain" and the "Delete domain in registry" buttons to sentence case, instead
- Updated the status message when clicking get status
- Modified existing test cases

## Setup
Navigate to the domain page in /admin

#### Satisfied acceptance criteria and met development standards

- [x] Met the acceptance criteria, or will meet them in a subsequent PR
- [x] Created/modified automated tests
- [x] Added at least 2 developers as PR reviewers (only 1 will need to approve)
- [x] Messaged on Slack or in standup to notify the team that a PR is ready for review
- [ ] Changes to “how we do things” are documented in READMEs and or onboarding guide
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.

#### Ensured code standards are met (Original Developer)

- [ ] All new functions and methods are commented using plain language
- [ ] Did dependency updates in Pipfile also get changed in requirements.txt?
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values

#### Validated user-facing changes (if applicable)

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)
- [ ] Add at least 1 designer as PR reviewer

#### Reviewed, tested, and left feedback about the changes

- [x] Pulled this branch locally and tested it
- [x] Reviewed this code and left comments
- [x] Checked that all code is adequately covered by tests
- [x] Made it clear which comments need to be addressed before this work is merged
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.

#### Ensured code standards are met (Code reviewer)

- [ ] All new functions and methods are commented using plain language
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values
- [ ] (Rarely needed) Did dependency updates in Pipfile also get changed in requirements.txt?

#### Validated user-facing changes as a developer

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Meets all designs and user flows provided by design/product
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers, the suggestion is to use ones that the developer didn't (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

**Note:** Multiple code reviewers can share the checklists above, a second reviewers should not make a duplicate checklist

#### Verified that the changes match the design intention

- [ ] Checked that the design translated visually
- [ ] Checked behavior
- [ ] Checked different states (empty, one, some, error)
- [ ] Checked for landmarks, page heading structure, and links
- [ ] Tried to break the intended flow

#### Validated user-facing changes as a designer

- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

### CODE REVIEW

**Review by Katherine-Osos — APPROVED** [review]

File: src/registrar/models/domain.py [hunk 1]
```diff
@@ -718,7 +718,7 @@ def map_epp_contact_to_public_contact(
 
         if not isinstance(contact, eppInfo.InfoContactResultData):
             raise ContactError("Contact must be of type InfoContactResultData")
-
+        # temp comment for push - will remove
```
Comment (erinysong) [inline src/registrar/models/domain.py:721]: Does this comment still need to be removed?

**Comment by erinysong** [comment]
LGTM! Just had one comment on the leftover comment mentioned above

**Review by erinysong — APPROVED** [review]
LGTM! Just had one comment on the leftover comment mentioned above

File: src/registrar/models/domain.py (hunk 1 above)
Comment (zandercymatics) [reply src/registrar/models/domain.py:721]: Nice catch, thank you!

**Review by rachidatecs — APPROVED** [review]

**Pull request merged**